content/client-lib-development-guide/features.textile

```diff
@@ -1636,6 +1636,7 @@ enum PushDeviceType:
   "mobile" // PDT1
   "tablet" // PDT1
   "desktop" // PDT1
+  "embedded" // PDT1
```

Comment:
I had previously suggested watch, which would probably be better as wearable in fact. However, looking at the docs, it appears you cannot send notifications to wearables, only to the mobile devices which in turn decide whether they display on the wearable or not, so leave as is.